src/shared/components/svg-icon.js

```diff
@@ -16,12 +21,24 @@ import propTypes from 'prop-types';
  */
 let iconRegistry = {};
 
+/**
+ * @typedef SvgIconProps
```

Comment:
Is the convention going forward: extended descriptions in typedef, no comments in `propTypes`? I'm down with that, just confirming...

Reply:
Yes, that was my suggestion at least.